Client/Application/WindowManager.swift

```diff
@@ -82,12 +82,14 @@ final class WindowManagerImplementation: WindowManager {
         // • If no saved windows (tab data), we generate a new UUID.
         // • If user has saved windows (tab data), we return the first available UUID
         //   not already associated with an open window.
-        // • If multiple window UUIDs are available, which is returned first is undefined.
-        //   TODO: [FXIOS-7929] ^ Temporary, part of ongoing multi-window work, eventually
+        // • If multiple window UUIDs are available, we currently return the first one
+        //   after sorting based on the uuid value.
+        //   TODO: [FXIOS-7929] This ^ is temporary, part of ongoing multi-window work, eventually
```

Comment:
Out of curiosity, how will we sort the windows in the future? Do we already know?

Reply:
> how will we sort the windows in the future? Do we already know?

This will probably be discussed further with product but the plan (AFAIK) is to track the notion of the "active" (or "primary") window, which will probably be whichever the user most recently interacted with. `WindowData` already has a property for this actually (`isPrimary`), there's just a bit more work still forthcoming to utilize that.